src/libvirt-dbus.js

```diff
@@ -1305,6 +1306,15 @@ export function attachIface({ connectionName, vmId, mac, permanent, hotplug, sou
     return attachDevice({ connectionName, vmId, permanent, hotplug, xmlDesc });
 }
 
+export function attachHostDevice({ connectionName, vmName, live, dev }) {
+    const source = getNodeDevSource(dev);
```

Comment:
This is only being used once, and attachHostDevice() is trivial. I suggest to just fold it in here, that makes the code easier to understand.

Reply:
It's being later used also for detachHostDevice in another commit